cime_config/tests.py

```diff
@@ -453,6 +453,15 @@
             )
     },
 
+    "e3sm_scream_v1_hires" : {
+        "time"  : "03:00:00",
+        "tests" : (
+            "SMS_D_Ln2.ne30_ne30.F2000SCREAMv1",
+            "SMS_D_Ln2.ne30_ne30.F2000-SCREAMv1-AQP1",
+            )
```

Comment:
Maybe have one of these tests include _D_ and the other not? ie have one debug build and one optimized build? Otherwise, not much to complain about!